### DISCUSSION TITLE
Error In Defect-Dojo installation using helm

### DISCUSSION THREAD

**Discussion by mrtechghost** [DefectDojo/django-DefectDojo#8268] [Q&A]
Hello Everyone,

I am trying to deploy defect-dojo using helm. When I am trying to run command helm install two new pods are not running properly .

1. defect-dojo-postgresql  :- error Init:ImagePullBackoff
2. defect-dojo-rabbitmq :- error :imagePullBackoff 

When try to get description it shows 0/3 nodes are availabe : 3 pod has unbound immediate persistentvolumeclaims.

also I have local repository for docker images still its trying to pull image from docker.io for bitmani-shell 

Can you please help me in above issues
![Error](https://github.com/DefectDojo/django-DefectDojo/assets/33446536/4ed4cb50-cc43-4516-b226-804aeb4266b2)
![error1](https://github.com/DefectDojo/django-DefectDojo/assets/33446536/452c9f06-9e64-48c3-a8b7-a65ac94777cc)
![error2](https://github.com/DefectDojo/django-DefectDojo/assets/33446536/57173a4f-0d49-4de1-8cb0-51dcc080cf3a)

**Comment by dsever** [comment]
Can you share your helm values?

And take a look how to overwrite default sidecar https://github.com/bitnami/charts/tree/main/bitnami/postgresql#volume-permissions-parameters with image from your repo

Describe the pod creation messages for the rabbit as well so we can identify the issue

**Reply by mrtechghost** [reply]
[Test.txt](https://github.com/DefectDojo/django-DefectDojo/files/11796965/Test.txt)
Hi dsever,

I have attached file for your reference.

**Reply by dsever** [reply]
```    fsGroup: 1001
    # any name
    name: media
    # could be emptyDir (not for production) or pvc
    type: pvc
    # in case if pvc specified, should point to the already existing pvc
    persistentVolumeClaim:
      # set to true to create a new pvc and if django.mediaPersistentVolume.type is set to pvc
      create: true
      name:
      size: 5Gi
      accessModes:
      - ReadWriteMany  # check KUBERNETES.md doc first for option to choose
      storageClassName:
```
Do you have really storage RW class by default (like NFS or some cloud privider specific)? 

For postgresql try t implement my previous proposal
